packages/edit-site/src/components/global-styles/font-families.js

```diff
@@ -39,6 +44,7 @@ function FontFamilies() {
 				.sort( ( a, b ) => a.name.localeCompare( b.name ) )
 		: [];
 	const hasFonts = 0 < customFonts.length || 0 < themeFonts.length;
+	const hasBaseFonts = baseFontFamilies?.theme?.length > 0;
```

Comment:
This conditional is not enough to know if there are fonts installed but not activated. Currenty, this only checks if the theme provides fonts that are not activated but  it doesn't check if there are custom fonts installed but they are not activated.

Reply:
You're right; it's likely possible that the only available fonts are custom ones that haven't been activated. I'll need to write an ad hoc function in order to add specific font conditions.